toolchain.txt

```diff
@@ -16,18 +16,10 @@ strip = '/tools/riscv/bin/riscv32-unknown-elf-strip'
 [properties]
 has_function_printf = false
 c_args = [
-  '-fvisibility=hidden',
-  '-g',
   '-mabi=ilp32',
   '-march=rv32imc',
   '-mcmodel=medany',
-  '-nostartfiles',
-  '-nostdlib',
-  '-Os',
-  '-static',
-  '-Wall',
-  '-Werror']
-c_link_args = ['-nostdlib']
+]
```

Comment:
Looks like this needs to be in the previous line, there's a parsing error in CI for this file.

Reply:
Thanks, this should be fixed!